docs/operation/common-problems.md

```diff
@@ -0,0 +1,95 @@
+# Common problems
+
+This document describes how to identify and resolve common Botkube problems that might occur.
+
+## Incompatible plugin API version
+
+**Symptoms**
+
+- Botkube agent Pod is restarting
+- In [Botkube agent logs](diagnostics.mdx#agent-logs), you see such entry:
+
+  ```text
+  while running application: while waiting for goroutines to finish gracefully: 1 error occurred:
+  	* while starting plugins manager: while creating executor plugins: Incompatible API version with plugin. Plugin version: 2, Client versions: [1]
+  ```
+
+**Debugging steps**
+
+- [Check Botkube agent version](diagnostics.mdx#agent-version).
+
+- [Check plugin repository version](diagnostics.mdx#check-configured-plugin-repositories).
+
+**Solution**
+
+In order to fix the problem, you need to make sure that the agent version is the same as the plugin repository version. For example, for agent image `ghcr.io/kubeshop/botkube:v1.4.0` you need to configure official plugin repository in version `v1.4.0`: `https://github.com/kubeshop/botkube/releases/download/v1.4.0/plugins-index.yaml`.
```

Comment:
Should we be sure about, version replacer replaces this version or will it be always `v1.4.0`?

Reply:
This is just an example, IMO we can leave it as it is, no need to have it always updated 👍